### PR TITLE
Complete Onboarding UI

### PR DESCRIPTION
**Pull request by greydnls**
This should be ready for CR now. The UI is all in place. I need to finish what happens after the login call succeed and also add in some error handling, but I think it's okay for now.

### CODE REVIEW

**Comment by ieatkillerbees** [comment]
I'd love you to walk me through this! I'm interested in learning how it works <3

**Comment by greydnls** [comment]
Wanna pair tonight to go over it?

File: src/Component/Onboarding/ColorPickerButton.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        ];
+
+        this.state = {
+            open: false,
+            color: this.props.child.color || _.sample(colors)
+        }
+    }
+
+    componentWillMount(){
+        this.props.onChange(this.props.index, this.state.color)
```
Comment (ieatkillerbees) [inline src/Component/Onboarding/ColorPickerButton.js]: This should only be called if `this.props.child.color` is null.

File: src/styles/createContext.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import { create, SheetsRegistry } from 'jss';
+import preset from 'jss-preset-default';
+import { createMuiTheme } from 'material-ui/styles';
+import { purple, green } from 'material-ui/colors';
+import createGenerateClassName from 'material-ui/styles/createGenerateClassName';
```
Comment (ieatkillerbees) [inline src/styles/createContext.js]: you don't need this file.

**Review by ieatkillerbees — APPROVED** [review]

**Review by ieatkillerbees — APPROVED** [review]

**Pull request merged**